docs/product/shortcuts.md

```diff
@@ -5,70 +5,100 @@ category: product
 tags: [product,ui]
 ---
 
-### The <kbd>meta</kbd> key.
-The <kbd>meta</kbd> key was introduced to make the shortcuts consistent across platforms. It is defined as <kbd>command</kbd> on macOS, and as <kbd>ctlr</kbd> on Windows and Linux.
+## General Assumptions
 
-### Graph Editor
+#### The <kbd>meta</kbd> key.
+The <kbd>meta</kbd> key was introduced to make the shortcuts consistent across platforms. 
+It is defined as <kbd>command</kbd> on macOS, and as <kbd>ctlr</kbd> on Windows and Linux.
+
+#### Keyboard-only Workflow
+The GUI and all shortcuts were designed in a way to allow both efficient mouse-only as well as
+keyboard-only workflows. In most cases, there is a relation between mouse and keyboard shortcuts,
+namely, the `left-mouse-button` corresponds to `enter`. For example, stepping into a node is done
+by either double clicking the node, or just pressing the enter key.
```

Comment:
This is a very important statement and I want us to keep it in our minds when working on the product. The fact that currently you cannot select nodes by keyboard shortcuts (arrows) or you cannot do sometinh with keyboard only is a very bad state, and we should improve it over the time (this PR improves it already). Of course, mouse / touch are amazing, but you. sohuld be able to work with keyboard only, which combined with a good auto-layouter could give you amazing workflow speed.

Reply:
But still, the docs does not say truth, and for me it even worse. Please rephrase it so we "tend" to make application for both mouse-and-keyboard and keyboard-only workflows. BTW we do not support "mouse only" workflow, because you still need to use modifier on your keyboard when using mouse.

The ultimate goals should be written in tasks, but in product docs we should have the current state, or just about-to-be-done features.